root/usr/libexec/nethserver/smbads

```diff
@@ -76,8 +76,8 @@ sub init_keytabs()
 
     # Initialize the system keytab, by adding required
     # primaries. 'cifs' is always added:
-    my $exitCode = system(qw(/usr/bin/net ads keytab add_update_ads -k),
-                          keys %{{map { (map { $_ => 1 } 'cifs', @{$_->{primary}}) } @ktConfigs}}
+    my $exitCode = system(qw(/usr/bin/net ads keytab add -k cifs),
+                          keys %{{map { (map { $_ => 1 } @{$_->{primary}}) } @ktConfigs}}
```

Comment:
I have no idea what this code does.

Reply:
It creates a list of unique elements with the collected values of `KrbPrimaryList` props. `cifs` is now always added, but I could check if it can be removed completely because Samba may add it automagically.